boards/Kconfig

```diff
@@ -1364,13 +1364,21 @@ config ARCH_BOARD_SABRE_6QUAD
 		board featuring the iMX 6Quad CPU.
 
 config ARCH_BOARD_SAMA5D2_XULT
-	bool "Atmel SAMA5D2 Xplained Ultra development board"
+bool "Atmel SAMA5D2 Xplained Ultra development board"
+depends on ARCH_CHIP_ATSAMA5D27
+        select ARCH_HAVE_LEDS
+        select ARCH_HAVE_BUTTONS
+        select ARCH_HAVE_IRQBUTTONS
+---help---
+The port of NuttX to the Atmel SAMA5D2-XULT development board
+
```

Comment:
The above does not follow the indentation rules that we use in Kconfig files.  Need a TAB before bool and depends and ---help--- and two TABs before "The port of NuttX"

Reply:
@patacongo Will fix.